### PR TITLE
UI Maintenance, May 2024

### PR DESCRIPTION
**Pull request by fadidurah**
Various adjustments to MSAL-only and MSAL/Broker Tests. Some tests were removed to be ran manually as part of release, some tests were adjusted, and some test files were removed corresponding to tests that have been migrated to integration testing. Changes in common were taken in to fix some broker automation.

Common: https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/2406

### CODE REVIEW

**Review by melissaahn — APPROVED** [review]

File: msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/msalonly/basic/TestCase497038.java [hunk 1]
```diff
@@ -50,6 +51,7 @@
 // https://identitydivision.visualstudio.com/DefaultCollection/DevEx/_workitems/edit/497038
 @RetryOnFailure
 @RunOnAPI29Minus("Consent Page")
+@Ignore("This one is failing because Azure Sample is 'disabled'")
```
Comment (rpdome) [inline msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/msalonly/basic/TestCase497038.java]: What do you mean by this?

File: msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/broker/nonjoined/TestCase1561158.java [hunk 2]
```diff
@@ -1,60 +0,0 @@
-//  Copyright (c) Microsoft Corporation.
```
Comment (rpdome) [inline msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/broker/nonjoined/TestCase1561158.java:1]: why are we removing these tests as opposed to ignoring them? Are they impossible to fix?

File: msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/broker/msa/TestCase2637829.kt [hunk 3]
```diff
@@ -45,10 +45,10 @@ import java.util.*
 @SupportedBrokers(brokers = [BrokerHost::class])
 @LocalBrokerHostDebugUiTest
 @RetryOnFailure
-class TestCase2637829 : AbstractMsaBrokerTest() {
+class TestCase2637829And2637846 : AbstractMsaBrokerTest() {
```
Comment (rpdome) [inline msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/broker/msa/TestCase2637829.kt:48]: Are these two tests similar? (in which case... we should probably just kill one of them from the test suite)

File: msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/msalonly/basic/TestCase497038.java (hunk 1 above)
Comment (fadidurah) [reply msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/msalonly/basic/TestCase497038.java]: This test has been rewritten both in ado and in this file.

File: msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/broker/nonjoined/TestCase1561158.java (hunk 2 above)
Comment (fadidurah) [reply msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/broker/nonjoined/TestCase1561158.java:1]: Most of these are now integration tests that pedro created. this one wasn't a valid scenario

File: msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/broker/msa/TestCase2637829.kt (hunk 3 above)
Comment (fadidurah) [reply msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/broker/msa/TestCase2637829.kt:48]: Both test cases are covered in this test file

Comment (rpdome) [reply msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/broker/msa/TestCase2637829.kt:48]: I meant perhaps we can just get rid one of the cases. Not a blocker though.

File: msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/msalonly/basic/TestCase497038.java (hunk 1 above)
Comment (rpdome) [reply msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/msalonly/basic/TestCase497038.java]: I mean I was confused with the "Azure Sample is disabled" part. not sure what that mean. 

Is Azure Sample itself broken? (even with manual test?, or is it just broken with automation?)

Comment (rpdome) [reply msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/msalonly/basic/TestCase497038.java]: The modified test seems to be using Msal Test app, so i'm not blocking on this.

**Review by rpdome — APPROVED** [review]

**Pull request merged**